src/Adapter/AbstractAdapter.php

```diff
@@ -4,5 +4,21 @@
 
 class AbstractAdapter implements AdapterInterface
 {
+    protected $trainingData;
+
+    public function getTrainingData()
+    {
+        return $this->trainingData;
+    }
+
+    public function train()
+    {
+        // TODO: Implement train() method.
+    }
+
+    public function predicte()
```

Comment:
I tend to prefer makePrediction() so we could have makeClassification() later...
What do you think ?